conda/recipes/pynvjitlink/meta.yaml

```
@@ -26,14 +30,14 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}
+    - {{ compiler('cuda') }} {{ cuda_version }}
```

Would think the version of the CUDA compiler would be added explicitly added to the hashes. This showed up in previous build before making this change. However now it doesn't show up. Though perhaps the hash does consider this (even though it is unlisted). For example this is [what CI shows]( https://github.com/rapidsai/pynvjitlink/actions/runs/10224784747/job/28292989319?pr=97#step:7:700 ):

```json
INFO :: The inputs making up the hashes for the built packages are as follows:
{
  "pynvjitlink-0.3.0-py312h127ddad_0.tar.bz2": {
    "recipe": {
      "c_compiler": "gcc",
      "c_compiler_version": "11",
      "c_stdlib": "sysroot",
      "c_stdlib_version": "2.17",
      "cuda_compiler": "cuda-nvcc",
      "cxx_compiler": "gxx",
      "cxx_compiler_version": "11",
      "target_platform": "linux-aarch64"
    }
  }
}
```

---

Let’s use cuda-version to constrain this instead of a manual pin.
```suggestion
    - {{ compiler('cuda') }}
    - cuda-version {{ cuda_version }}
```

---

Yep can do. Added below: https://github.com/rapidsai/pynvjitlink/pull/97/commits/c944a3aa97371ad6c8dacbd99cb2846e39ff4aca

Had tried a bunch of variations of this. However the hash observation above remained unchanged

---

Fixed this by switching to a variant file as described in related comments